src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -658,6 +658,7 @@
     <Compile Include="System\Threading\ClrThreadPool.ThreadCounts.cs" />
     <Compile Include="System\Threading\ClrThreadPool.WorkerThread.cs" />
     <Compile Include="System\Threading\EventWaitHandle.Unix.cs" />
+    <Compile Include="System\Threading\HighPerformanceCounter.Unix.cs" />
```

Comment:
Could put this directly in System or System\Diagnostics where Stopwatch would be